### ISSUE TITLE
alt text is missing for images - General

### ISSUE THREAD

**Issue by Aswathipanda** [AbhiramPm/NEOFILMS#68]
https://www.neofilmschool.com/uploads/workshop-faculty/bd92c63d1e668f749ec8cca7ab85eb90.jpg
